examples/counter_out/write_dig_pulse.py

```diff
@@ -0,0 +1,19 @@
+"""Example for generating digital pulse.
```

Comment:
I think being clear this is a single pulse is nice. People are often confused (fairly!) by our API when this example only makes a single pulse. They tend to expect this will do the Continuous pulses!

Change the example to `write_single_dig_pulse.py`.

```suggestion
"""Example for generating a single digital pulse.
```

Reply:
file is renamed and docstring is updated as per suggestion.